conf.py

```diff
@@ -41,7 +41,10 @@
 extensions = [
     'sphinx.ext.todo',
     'sphinx.ext.viewcode',
-    'sphinx.ext.graphviz'
+    'sphinx.ext.graphviz',
+     #For plantuml to work you need a plantuml command be on your path. This
+     # shall start plantuml.jar as: java -jar /path/to/plantuml.jar $@
+     'sphinxcontrib.plantuml'
```

Comment:
Unsure how this will work, I've tried Google it, but didn't find any obvious solution for
a) Enable it in Travis
b) Enable it at readthedocs.io

There were some things that seems to be worth trying.

Reply:
Hi,

there are the following dependencies:

1. graphviz 
2. java runtime environment
3. plamtUML
4. the sphinxcontrib-plantuml PyPI package

On Ubuntu 1-3 can be installed with `sudo apt install plantuml `. 4 can be installed with `pip3 install sphinxcontrib-plantuml`.

#### RTD
The [RTD build container](https://github.com/readthedocs/readthedocs-docker-images/blob/f03a67c16bfd879b4d61729fb6aebc793540fae1/Dockerfile#L98) has planuml and its dependencies installed.
[Additional python packages](https://docs.readthedocs.io/en/stable/faq.html#my-documentation-requires-additional-dependencies)  PyPI packages can be set-up with a configuration file, but TS documentation is using plantuml, build fine and we don't have the mentioned extra config. Although we have a `requirements.txt` file, which might be taken up by RTD automatically. This is not mentioned in the documentation though.

#### Travis
I don't know much about the environment used here. `sudo apt install plantuml` and `pip install sphinxcontrib-plantuml` should get everything needed, if the environment is ubuntu based.

Reply:
@gyuri-szing: thanks, that gives a couple of things to try. Having said that, I'm unable to do it right away, so the PR will have to be on hold for a little longer until I've found some time for try it out. Shouldn't be that long though ...

Reply:
@jbech-linaro  did you had any time to look at this?

Reply:
I'll give it  a shot.

Reply:
> I'll give it a shot.

Thanks Jens!